examples/yarn-pnp/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Let's keep the example as small as possible & remove the `.vscode` folder

Reply:
will do